packages/by-name/microsoft/genpolicy/genpolicy_msft_settings_coordinator.patch

```diff
@@ -12,10 +12,6 @@ index 7d35862a..4eacc7cd 100644
              ]
          },
          "CopyFileRequest": [
-@@ -322,4 +323,4 @@
-         "UpdateEphemeralMountsRequest": false,
-         "WriteStreamRequest": false
-     }
```

Comment:
Why is this necessary?

Reply:
I think that was an accident, will revert that part.

Reply:
Nvm, I did some digging, I removed that part because it is added by default in the new patch. I didn't make the settings changes into a "settings patch" since, following the PR in msft.kata-containers, they have to be there for all settings instead of being some specialization.